src/docs/development/ui/layout/index.md

```diff
@@ -1,1191 +1,52 @@
-          children: [
-            Icon(Icons.kitchen, color: Colors.green[500]),
-            Text('PREP:'),
-            Text('25 min'),
-          ],
-        ),
-        Column(
-          children: [
-            Icon(Icons.timer, color: Colors.green[500]),
-            Text('COOK:'),
-            Text('1 hr'),
-          ],
-        ),
-        Column(
-          children: [
-            Icon(Icons.restaurant, color: Colors.green[500]),
-            Text('FEEDS:'),
-            Text('4-6'),
-          ],
-        ),
-      ],
-    ),
-  ),
-);
-```
-
-The `leftColumn` variable contains the ratings and icons rows, as well as the
-title and text that describes the Pavlova:
-
-<?code-excerpt "layout/pavlova/lib/main.dart (leftColumn)" replace="/leftColumn/[!$&!]/g"?>
-```dart
-final [!leftColumn!] = Container(
-  padding: EdgeInsets.fromLTRB(20, 30, 20, 20),
-  child: Column(
-    children: [
-      titleText,
-      subTitle,
-      ratings,
-      iconList,
-    ],
-  ),
-);
-```
-
-The left column is placed in a `Container` to constrain its width.
-Finally, the UI is constructed with the entire row (containing the
-left column and the image) inside a `Card`.
-
-The [Pavlova image][] is from [Pixabay][].
-You can embed an image from the net using `Image.network()` but,
-for this example, the image is saved to an images directory in the project,
-added to the [pubspec file,]({{examples}}/layout/pavlova/pubspec.yaml)
-and accessed using `Images.asset()`. For more information, see
-[Adding assets and images](/docs/development/ui/assets-and-images).
-
-<?code-excerpt "layout/pavlova/lib/main.dart (body)"?>
-```dart
-body: Center(
-  child: Container(
-    margin: EdgeInsets.fromLTRB(0, 40, 0, 30),
-    height: 600,
-    child: Card(
-      child: Row(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Container(
-            width: 440,
-            child: leftColumn,
-          ),
-          mainImage,
-        ],
-      ),
-    ),
-  ),
-),
-```
-
-{{site.alert.tip}}
-  The Pavlova example runs best horizontally on a wide device, such as a tablet.
-  If you are running this example in the iOS simulator, you can select a
-  different device using the **Hardware > Device** menu. For this example, we
-  recommend the iPad Pro. You can change its orientation to landscape mode using
-  **Hardware > Rotate**. You can also change the size of the simulator window
-  (without changing the number of logical pixels) using **Window > Scale**.
+{{site.alert.note}} Dart is the coding language that Flutter developers use to create their apps. What's great about Dart is that it's object oriented and widget based. For example, *images*, *icons*, and *text* are widgets. *Columns*, *rows*, and *grids* also are  widgets. Widgets compile the ins and outs of Flutter, and, because Flutter is object oriented, you not only can create, but also collaborate with other Flutter developers to fix technical issues more easily.   
```

Comment:
The phrase "because Flutter is object oriented, ..., you can also collaborate with other..."
This sentence doesn't make sense to me.